templates/tfengine/components/cicd/manual/main.tf

```diff
@@ -68,6 +68,7 @@ locals {
   cloudbuild_devops_roles = [
     # Enable Cloud Build SA to list and enable APIs in the devops project.
     "roles/serviceusage.serviceUsageAdmin",
+    "roles/secretmanager.viewer",
```

Comment:
You might need admin permission. I got an error with just viewer permission with a full clean deployment. But it also depends on if CICD is actually filling values for some secrets.

Reply:
Or at least `roles/secretmanager.secretAccessor`.